tests/suites/test_suite_pkwrite.data

```diff
@@ -1,47 +1,95 @@
 Public key write check RSA
 depends_on:MBEDTLS_RSA_C:MBEDTLS_BASE64_C
-pk_write_pubkey_check:"data_files/server1.pubkey"
+pk_write_pubkey_check:"data_files/server1.pubkey":0
+
+Public key write check RSA (DER)
+depends_on:MBEDTLS_RSA_C:MBEDTLS_BASE64_C
```

Comment:
The DER tests don't need `BASE64_C` (or PEM). That's one of the main reason we're adding them ;)